src/community/label_propagation.jl

```diff
@@ -4,13 +4,16 @@ Community detection using the label propagation algorithm (see [Raghavan et al.]
 `maxiter`: optional, maximum iterations
 return : array containing vertex assignments, the range of the output as 1:countdistinct(membership)
 """
-function label_propagation(g::SimpleGraph; maxiter=1000)
+function label_propagation(g::SimpleGraph; maxiter=1000, verbose=false)
```

Comment:
I'd reconsider making `maxiter` (at least, and possibly `verbose`) regular arguments with default values as opposed to kwargs. kwargs represent inefficiencies.